src/txn/txn_recover.c

```diff
@@ -1152,8 +1159,10 @@ __wt_txn_recover(WT_SESSION_IMPL *session, const char *cfg[])
             eviction_started = true;
         }
 
-        __wt_verbose_multi(session,
-          WT_DECL_VERBOSE_MULTI_CATEGORY(((WT_VERBOSE_CATEGORY[]){WT_VERB_RECOVERY, WT_VERB_RTS})),
+        __wt_verbose_level_multi(session,
+          WT_DECL_VERBOSE_MULTI_CATEGORY(
+            ((WT_VERBOSE_CATEGORY[]){WT_VERB_RECOVERY, WT_VERB_RECOVERY_PROGRESS, WT_VERB_RTS})),
```

Comment:
Should `WT_VERB_RECOVERY_ALL` include `WT_VERB_RTS` so we can simply use `WT_VERB_RECOVERY_ALL`?

Reply:
Hmm, No. RTS has a specific log checker that verifies all the log messages that are logged with WT_VERB_RTS. If we include recovery messages also as part of the WT_VERB_RTS, we need to add log handling for them also.